contrib/completions/bash/oc

```diff
@@ -576,8 +576,13 @@ _oc_adm_catalog_build()
     flags+=("--from-dir=")
     two_word_flags+=("--from-dir")
     local_nonpersistent_flags+=("--from-dir=")
+    flags+=("--icsp-file=")
```

Comment:
Pls split the generated bits like this to a separate commit.

Reply:
done